INFO     backend.layers.business.business:business.py:511 {'message': 'ingesting dataset', 'collection_version_id': 8db5caff-0781-4188-9955-a0ede9c0bfd2, 'url': 'http://fake.url', 'current_dataset_version_id': None}
INFO     backend.layers.business.business:business.py:511 {'message': 'ingesting dataset', 'collection_version_id': 8db5caff-0781-4188-9955-a0ede9c0bfd2, 'url': None, 'current_dataset_version_id': 085af7ec-9104-4af4-87c9-d8594ef50af6}
INFO     processing:process_logic.py:45 Updating processing status
INFO     processing:process_logic.py:45 Updating processing status
INFO     processing:logger.py:26 Start create_artifact
INFO     processing:process_logic.py:45 Updating processing status
INFO     processing:process_logic.py:91 Uploaded [4311e9cf-513c-4850-8b06-d318e1cdcaee/raw.h5ad] to s3://artifact_bucket/4311e9cf-513c-4850-8b06-d318e1cdcaee/raw.h5ad
INFO     processing:process_logic.py:93 Updated database with raw_h5ad.
INFO     processing:process_logic.py:45 Updating processing status
INFO     processing:logger.py:28 Complete create_artifact
INFO     processing:process_logic.py:45 Updating processing status
INFO     processing:dataset_metadata_update.py:244 Main: Starting thread for h5ad update
INFO     processing:dataset_metadata_update.py:264 Main: Starting thread for rds update
ERROR    processing:dataset_metadata_update.py:302 Cannot find cxg artifact uri for 085af7ec-9104-4af4-87c9-d8594ef50af6.
INFO     processing:process_logic.py:45 Updating processing status
INFO     processing:process_logic.py:45 Updating processing status